migrate/migrations/20240212125328-alert-metadata.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+CREATE TABLE alerts_metadata (
+	alert_id integer NOT NULL,
+	id uuid DEFAULT gen_random_uuid() NOT NULL,
+	metadata jsonb,
+	CONSTRAINT alerts_metadata_alert_id_fkey FOREIGN KEY (alert_id) REFERENCES alerts(id) ON DELETE CASCADE,
+	CONSTRAINT alerts_metadata_pkey PRIMARY KEY (id, alert_id),
+	CONSTRAINT alerts_metadata_id UNIQUE (alert_id)
+);
+
+-- +migrate Down
+DROP TABLE alerts_metadata;
```

Comment:
Let's follow the same pattern we use for schedules

```suggestion
CREATE TABLE alert_data (
	alert_id BIGINT PRIMARY KEY REFERENCES alerts (id) ON DELETE CASCADE,
	metadata jsonb
);

-- +migrate Down
DROP TABLE alert_data;
```